app/assets/javascripts/discourse/app/templates/components/reviewable-field-category.hbs

```diff
@@ -1 +1 @@
-<CategoryChooser @value={{this.value}} @onChange={{action "onChange"}} />
+<CategoryChooser @value={{this.categoryId}} @onChange={{action "onChange"}} />
```

Comment:
I think this can be just:

```suggestion
<CategoryChooser @value={{this.categoryId}} @onChange={{this.onChange}} />
```

Reply:
O good point, I updated the component but forgot about the template.